donut/modules/permissions/helpers.py

```diff
@@ -0,0 +1,24 @@
+import flask
+from donut.modules.groups import helpers as groups
+
+
+def has_permission(user_id, permission_id):
```

Comment:
How will modules know which permission_id to call on? It should be some descriptor of the permission instead, either has_permissions(user_id, permission_name) or an ENUM of permissions stored in a file.

Reply:
If we make an enum that just duplicates what's already stored in the permissions table. How about storing a column for permission name and permission description, then making a function that filters on permission name?